packages/flutter/test/material/bottom_sheet_theme_test.dart

```diff
@@ -59,6 +59,7 @@ void main() {
       home: Scaffold(
         body: BottomSheet(
           onClosing: () {},
+          enableDrag: false,
```

Comment:
This indicates a breaking change.

It may be a breaking change worth making, since the only reason these tests passed previously is that they didn't exercise this behavior - but it still should follow our breaking change process (probably just needing a migration guide rather than a design doc).

I think this could be a non-breaking change if we just asserted when the user actually dragged, but that's an unpleasant developer experience.  It seems worth making this break.

We also should have a test that shows the assert happens, so that someone doesn't just remove the assert and see that all tests still pass :)